src/python/pants/backend/jvm/subsystems/rsc_native_image.py

```diff
@@ -0,0 +1,31 @@
+# coding=utf-8
+# Copyright 2019 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+from pants.backend.jvm.subsystems.zinc import _ZINC_COMPILER_VERSION
+from pants.binaries.binary_tool import NativeTool
+from pants.util.memo import memoized_method
+
+
+class RscNativeImage(NativeTool):
+  options_scope = 'rsc-native-image'
```

Comment:
So, although this is the first set of options that have been extracted from the jvm_compile Task for rsc, I think that it should not be the last.

I'd suggest just naming this `rsc`. Later on we can move the `register_jvm_tool` definitions here as well.